public/components/datasources/components/manage/accelerations/utils/acceleration_utils.tsx

```diff
@@ -80,12 +123,12 @@ export const getRefreshButtonIcon = () => {
   return 'inputOutput';
 };
 
-export const onRefreshButtonClick = (acceleration: any) => {
+export const onRefreshIconClick = (acceleration: CachedAcceleration) => {
   // TODO: send request to refresh
-  console.log('refreshing', acceleration.name);
+  console.log('refreshing', acceleration.indexName);
 };
```

Comment:
Is there still implementation work to be done on this?

Reply:
I will remove this. Good catch

Reply:
removed.